pubspec.yaml

```diff
@@ -34,7 +34,7 @@ dependencies:
 
   # native
   share_plus: ^7.0.2
-  url_launcher: ^6.0.3
+  url_launcher: ^6.1.11
```

Comment:
Is this still needed?

Reply:
Not sure. I guess i can try building without it.